### ISSUE TITLE
Some errors on begin

### ISSUE THREAD

**Issue by FrankWillianDettmer** [jennybc/googlesheets#390]
Hi, I'm trying to create a script to fetch google spreadsheets to connect to power bi based on this video: https://youtu.be/8lWR-_L52Qs
link step by step: https://itsalocke.com/blog/connect-to-google-sheets-in-power-bi-using-r/

When I use the instruction from step 2

"library(googlesheets)
token <- gs_auth(cache = FALSE)
gd_token()
saveRDS(token, file="googlesheets_token.rds")"

I have this return:

"> saveRDS(token, file="googlesheets_token.rds")
Error in saveRDS(token, file = "googlesheets_token.rds") : 
  token object not found
> "

Would you know how to help me?

**Comment by jennybc** [comment]
Hard to say but it looks like the `token <- gs_auth(cache = FALSE)` call was not successful, which then leads to the `token object not found` result.

Just so you know, googlesheets (this package) is being sunsetted very soon, in favour of a successor googlesheets4:

https://github.com/tidyverse/googlesheets4

Also, depending on what you're doing, a great deal of googlesheets's functionality (this package) can be accomplished with the already-released and actively-maintained googledrive package:

This is package is being sunsetted in favour of its replacement googlesheets4. I highly recommend planning new work around that, if at all possible.

https://github.com/tidyverse/googlesheets4

Just so you know, googlesheets (this package) is being sunsetted very soon, in favour of a successor googlesheets4:

https://github.com/tidyverse/googlesheets4

Also, depending on what you're doing, a great deal of googlesheets's functionality (this package) can be accomplished with the already-released and actively-maintained googledrive package:

https://googledrive.tidyverse.org

Anything that operates at the whole file level can be done with googledrive and, even here, has always involved calls to the Drive API instead of Sheets.

So, in terms of developing your workflow, I recommend you look towards those packages as dependencies and not googlesheets.

**Comment by FrankWillianDettmer** [comment]
Hello, my goal is to fetch information from a particular google spreadsheet to put on power bi.

Im very new to use of R, and when i try to install gg4 i get this message: "package‘ googlesheets4 ’is not available (for R version 3.5.3)"

On https://github.com/tidyverse/googlesheets4 says "**## NO, NO YOU CANNOT
## install.packages("googlesheets4")**", im very confuse about what to do...

**Comment by jennybc** [comment]
googlesheets4 is coming soon-is to CRAN, at which point `install.packages()` will work.

For now, you would need to follow the second set of instructions.

First, install the devtools package from CRAN:

``` r
install.packages("devtools")
```

Then install googlesheets4 from GitHub:

``` r
And the development version from GitHub with:
devtools::install_github("tidyverse/googlesheets4")
```

As you encounter more usage questions (that's how I'd describe this, as opposed to bug or feature request), you might get a lot out of lurking and asking questions on RStudio's community site. It's very friendly:

https://community.rstudio.com
